framework/db/ColumnSchemaBuilder.php

```diff
@@ -105,6 +105,17 @@ public function defaultValue($default)
     }
 
     /**
+     * Specify the default DB Expression for the column.
```

Comment:
better: `Specify the default value for the column in terms of a [[Expression|SQL expression]].`